index.html

```diff
@@ -125,13 +125,20 @@
         every transaction: the merchant (e.g., an online web store), the buyer
         represented by the <a>user agent</a> (e.g., the user buying from the
         online web store), and the <dfn data-lt=
-        "payment method|payment methods">Payment Method</dfn> (e.g., credit
+        "payment method|payment methods">payment method</dfn> (e.g., credit
         card). Information necessary to process and confirm a transaction is
-        passed between the <a>Payment Method</a> and the merchant via the
+        passed between the <a>payment method</a> and the merchant via the
         <a>user agent</a> with the buyer confirming and authorizing as
         necessary across the flow.
       </p>
       <p>
+        The details of how to fulfill a payment request for a given <a>payment
+        method</a> are handled by <dfn data-lt=
+        "payment app|payment apps">payment apps</dfn>. In this specification,
+        these details are left up to the <a>user agent</a>, but future
+        specifications may expand on the processing model in more detail.
```

Comment:
I think this is fine for now - but we should have at least 1 base thing here (probably "basic-card").